app/src/main/java/org/liberty/android/fantastischmemo/ui/StatisticsScreen.java

```diff
@@ -246,6 +257,8 @@ public Chart generateChart(BarData data) {
             chart.getXAxis().setTextColor(Color.WHITE);
             chart.getAxisLeft().setTextColor(Color.WHITE);
             chart.getAxisRight().setTextColor(Color.WHITE);
+            chart.getAxisLeft().setValueFormatter(new ChartYAxisValueFormatter());
```

Comment:
I think you can have a static variable for the formatter instead of creating a instance each time you use it.

privat static ChartYAxisValueFormatter yAxisValueFormatter = new ChartYAxisValueFormatter();

and use it in all places. I think it should work and saves a bit memory